src/main/src/main/java/org/geoserver/data/util/IOUtils.java

```diff
@@ -54,10 +54,18 @@ private IOUtils() {
      * @throws IOException
      */
     public static void copy(InputStream from, File to) throws IOException {
-        OutputStream out = null;
-        try {
-            out = new FileOutputStream(to);
-
+        copy(from, new FileOutputStream(to));
+    }
+    
+    /**
+     * Copies the provided input stream onto an outputstream
+     * 
+     * @param from
+     * @param to
+     * @throws IOException
+     */
+    public static void copy(InputStream from, OutputStream out) throws IOException {
```

Comment:
@NielsCharlier this is an unusual division of resource management responsibility. My expectation would be that callers are responsible for closing streams they open. Here that responsibility is being passed to the callee. An example of why this is problematic is that it prevents the use of try-with-resources and forces the use of null tests. Could this be refactored? Or as an existing API is it to hard to change the semantics? Perhaps @aaime can comment as the ```@author```.